### PR TITLE
Switch location for goautoneg vendored code

### PR DESCRIPTION
**Pull request by dims**
Move to github.com/munnerz/goautoneg as bitbucket is flaky!

Change-Id: Iaa6e964ef0d6f308eea59bcc6f365ecd7dbf0784

**What type of PR is this?**

/kind flake

**What this PR does / why we need it**:

bitbucket is unreliable. same code has been in @munnerz 's repo for a while. let's switch to that

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by krzyzacy** [comment]
qq: how does it work with vendor deps? assume we bump vendor next time it will come back?

**Comment by dims** [comment]
Need this to merge first - https://github.com/kubernetes/kube-openapi/pull/126

File: api/openapi-spec/swagger.json [hunk 1]
```diff
@@ -78865,6 +78865,7 @@
   "definitions": {
    "io.k8s.api.admissionregistration.v1alpha1.Initializer": {
     "description": "Initializer describes the name and the failure policy of an initializer, and what resources it applies to.",
+    "type": "object",
```
Comment (liggitt) [inline api/openapi-spec/swagger.json]: didn't expect these changes either

Comment (liggitt) [reply api/openapi-spec/swagger.json]: nevermind, I see the version bump

File: vendor/k8s.io/kube-openapi/pkg/generators/openapi.go [hunk 2]
```diff
@@ -336,6 +336,7 @@ func (g openAPITypeWriter) generate(t *types.Type) error {
 		}
 		g.Do("return $.OpenAPIDefinition|raw${\nSchema: spec.Schema{\nSchemaProps: spec.SchemaProps{\n", args)
 		g.generateDescription(t.CommentLines)
+		g.Do("Type: []string{\"object\"},\n", nil)
```
Comment (sttts) [inline vendor/k8s.io/kube-openapi/pkg/generators/openapi.go]: this was an accident. Certainly has nothing to do with the actual PR.

**Pull request merged**